portality/templates/base.html

```diff
@@ -40,7 +40,7 @@
 
     <!-- get the app's css -->
     <link rel="stylesheet" type="text/css" href="/static/css/doaj.css?v={{config.get('DOAJ_VERSION')}}">
-
+    <script type="text/javascript" src="/static/js/contact.js?v={{config.get('DOAJ_VERSION')}}"></script>
```

Comment:
This should go in the `contact.html` template, in a section that looks like this:

```
{% block extra_js_bottom %}
<script type="text/javascript" src="/static/js/contact.js?v={{config.get('DOAJ_VERSION')}}"></script>
{% endblock extra_js_bottom %}
```

Just to clarify why:
* loading JS should be done as late in the page load as possible, so that it doesn't interrupt the page render.  That's why we load all our JS at the end, and much of it is in a template called `_js_includes.html`.
* We don't need this feature except on a contact page, so there's no point loading it on all the other pages, which is what would happen if you put it in the base template.

Reply:
In fact, defined as it is right now, it doesn't work at all because jquery isn't loaded until later :)

Reply:
Actually, I just looked, and it's already defined in contact.html, so you can just omit this.